packages/web-app-files/src/components/SideBar/Shares/FileLinks.vue

```diff
@@ -63,7 +63,40 @@
         </li>
       </oc-list>
       <div v-if="links.length > 3" class="oc-flex oc-flex-center">
-        <oc-button appearance="raw" @click="toggleLinkListCollapsed" v-text="collapseButtonTitle" />
+        <oc-button appearance="raw" @click="toggleLinkListCollapsed">
+          <span v-text="collapseButtonTitle" />
+          <oc-icon :name="collapseButtonIcon" fill-type="line" />
+        </oc-button>
+      </div>
+      <div v-if="indirectLinks.length">
+        <hr class="link-separator oc-my-m" />
+        <h3 class="oc-text-bold oc-m-rm oc-text-initial">
+          <span v-text="indirectLinksHeading" />
+          <oc-contextual-helper v-if="helpersEnabled" v-bind="indirectLinkHelp" />
+        </h3>
+        <oc-list v-if="displayIndirectLinks.length" class="oc-overflow-hidden oc-my-m">
```

Comment:
Why the new computed prop? Using `!indirectLinkListCollapsed` for the `v-if` here and then iterating over the `indirectLinks` for the `li`s does the same afaict.

Reply:
Taken care of, thx